app/gosns/create_topic_test.go

```diff
@@ -0,0 +1,105 @@
+package gosns
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/Admiral-Piett/goaws/app"
+	"github.com/Admiral-Piett/goaws/app/fixtures"
+	"github.com/Admiral-Piett/goaws/app/interfaces"
+	"github.com/Admiral-Piett/goaws/app/models"
+	"github.com/Admiral-Piett/goaws/app/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateTopicV1_success(t *testing.T) {
+	app.CurrentEnvironment = fixtures.LOCAL_ENVIRONMENT
```

Comment:
Do we uniform how to prepare `CurrentEnvironment` with `fixtures...` or `conf.LoadYamlConfig(...)`?
If they are not going to be standardized, it would be nice to have a guideline for which one to use.

Reply:
They are different because it depends on what you need each time.  Doing it like this sets defaults and what-not, but doesn't build out the resources (queues/topics).  Doing it with `conf.LoadYamlConfig` does build out resources.

Reply:
OK, let's consider on each API 👍